.devcontainer/post-install.sh

```diff
@@ -1,3 +1,4 @@
 #!/bin/bash
 poetry install
 poetry shell
+pre-commit install
```

Comment:
Still needs to be updated here

```suggestion
poetry run pre-commit install
```